R/Intervention_file_generator.R

```diff
@@ -0,0 +1,147 @@
+
+generate_intervention_file <- function() {
+  #######################
+  ## LLINs
+  ## At each timepoint, x% of the population receive a new net.
+  ## If they have an existing net it is replaced.
+
+  LLIN_years <- c() # c(2002, 2005, 2008, 2011, 2015, 2018, 2021, 2024, 2027, 2030, 2033, 2036, 2039, 2042)

```

Comment:
Perfect, I have made an [issue](https://github.com/mrc-ide/vivax/issues/3) for this so that we action it. I've added it to the [GTS milestone](https://github.com/mrc-ide/vivax/milestone/1)

I've also created an issue for [c++ RNGs](https://github.com/mrc-ide/vivax/issues/2). If that's needed for GTS, please put it on the GTS milestone too and I'll have a look at it sooner rather than later.